iroha/src/lib.rs

```diff
@@ -112,7 +123,7 @@ impl Iroha {
                             sumeragi
                                 .lock()
                                 .await
-                                .sign(transactions)
+                                .sign_transactions(transactions)
```

Comment:
if we have only one function which starts with `sign` we can stay with this naming. because right now it reads as - sign_transactions_transactions.

Reply:
There is also a function `sign_block` in Sumeragi.